packages/dialog/theme/material/vaadin-dialog-styles.js

```diff
@@ -10,29 +10,94 @@ const dialogOverlay = css`
     max-width: 560px;
     min-width: 280px;
     -webkit-tap-highlight-color: transparent;
+    border-radius: 8px;
```

Comment:
I'd like to extract this change to separate PR as well, to have a separate entry in the release notes.